plugin/src/main/java/net/dzikoysk/funnyguilds/config/transformer/formatting/NumberRangeFormattingTransformer.java

```diff
@@ -0,0 +1,60 @@
+package net.dzikoysk.funnyguilds.config.transformer.formatting;
+
+import eu.okaeri.configs.schema.GenericsPair;
+import eu.okaeri.configs.serdes.BidirectionalTransformer;
+import eu.okaeri.configs.serdes.SerdesContext;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.dzikoysk.funnyguilds.config.range.formatting.NumberRangeFormatting;
+
+public class NumberRangeFormattingTransformer extends BidirectionalTransformer<String, NumberRangeFormatting> {
```

Comment:
Czym się różni number range od integer/double, ten kod jest prkatycznie identyczny we wszystkich klasach

Reply:
W sumie fakt... Zostawiłem teraz po prostu NumberRange i tyle